File: src/main/test/sample/model/PackageChangerTest.java
package sample.model;

import org.junit.Test;

import java.io.File;
import java.io.IOException;

import static org.junit.Assert.*;

public class PackageChangerTest {

    @Test
    public void rewriteFile() throws IOException {
        File file = new File("d:\\1");
        PackageChanger packageChanger =
                new PackageChanger(file,
                        "iceblood.computercomponents","com.bla");
        packageChanger.rewriteFile(file);
    }

    @Test
    public void searchFile() throws IOException {
        File file = new File("d:\\1");
        PackageChanger packageChanger =
                new PackageChanger(file,
                        "iceblood.computercomponents","com.bla");
        packageChanger.searchFile(file);
    }
}

File: src/test/java/sample/model/PackageChangerTest.kt
package sample.model

import org.junit.Test
import sample.build.PackageChanger
import java.io.File
import java.io.IOException

class PackageChangerTest {


    private val projectPath = "d:\\1"

    @Test
    @Throws(IOException::class)
    fun rewriteFile() {
        val file = File(projectPath)
        val packageChanger = PackageChanger(file,
                "iceblood.computercomponents", "com.bla")
        packageChanger.rewriteFile(file)
    }

    @Test
    @Throws(IOException::class)
    fun searchFile() {
        val file = File(projectPath)
        val packageChanger = PackageChanger(file,
                "iceblood.computercomponents", "com.bla")
        packageChanger.searchFile(file)
    }
}